src/bundles/peers.js

```diff
@@ -9,6 +9,43 @@ const bundle = createAsyncResourceBundle({
   checkIfOnline: false
 })
 
+bundle.selectTableData = state => {
+  const peers = state.peers.data
+
+  return peers && peers.map((peer, idx) => ({
+    'id': peer.peer.toB58String(),
+    'address': peer.addr.toString(),
+    'location': 'New York, United States'
```

Comment:
Why's New York here?

Reply:
Oops, that selector is not being used, I made a new below. I'll commit a fix.